coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = vars(result)
         try:
             if len(result.affected_code) == 0:
+                args['affected_code'] = None
                 print(format_str.format(file=None,
                                         line=None,
                                         end_line=None,
                                         column=None,
                                         end_column=None,
                                         severity_str=severity_str,
-                                        **result.__dict__))
+                                        **args))
```

Comment:
`args`(Considering you have not changed the `args` variable which is passed to this enclosing function) will print the same information on every iteration of the for loop and of-course `args` may not contain what we want.